src/admin/content/content.services.ts

```diff
@@ -89,28 +89,31 @@ export const insertContent = async (
 
 export const updateContent = async (
 	contentItem: Partial<Avo.Content.Content>,
+	initialContentBlocks: ContentBlockSchema[],
 	contentBlockConfigs: ContentBlockConfig[],
-	triggerContentInsert: MutationFunction<Partial<Avo.Content.Content>>
+	triggerContentUpdate: MutationFunction<Partial<Avo.Content.Content>>
 ): Promise<Partial<Avo.Content.Content> | null> => {
 	try {
-		const response = await triggerContentInsert({
+		const response = await triggerContentUpdate({
 			variables: {
 				contentItem,
 				id: contentItem.id,
 			},
 			update: ApolloCacheManager.clearContentCache,
 		});
-		const insertedContent = get(response, 'data', null);
+		const updatedContent = get(response, 'data', null);
 
 		if (contentBlockConfigs && contentBlockConfigs.length) {
-			// TODO: Add logic for:
-			// - inserting content-blocks
-			// - updating content-blocks
-			// - deleting content-blocks
+			await updateContentBlocks(
+				contentItem.id as number,
+				initialContentBlocks,
+				contentBlockConfigs
+			);
 		}
 
-		if (!insertedContent) {
+		if (!updatedContent) {
 			console.error('Content update returned empty response', response);
```

Comment:
zou dit niet beter aangepast worden naar 
```typescript
throw new Error('Content update returned empty response')
```
en dan in de component die deze ui gebruikt een toast tonen.

Misschien kan je ook dezelfde helper gebruiken voor error objecten waar je een innerException en additionalInfo kan aan meegeven:
```typescript
export default class CustomError {
  public message: string;
  public innerException: any | null;
  public additionalInfo: any | null;
  public identifier: string = Math.random().toString().substring(2, 9);
  public name: string = 'Error';
  public stack: string;
  public timestamp: string = new Date().toISOString();

  constructor(
      message: string = 'Something went wrong',
      innerException: any = null,
      additionalInfo: any = null) {
    this.message = message;
    this.innerException = innerException;
    this.additionalInfo = additionalInfo;
    Error.captureStackTrace(this, this.constructor);

    if (innerException && typeof innerException.stack === 'string') {
      this.stack = innerException.stack;
    } else {
      this.stack = new Error().stack || '';
    }
  }

  public toString(): string {
    return JSON.stringify(
        this,
        (key, value) => {
          if (key === 'request') {
            // Avoid huge request object in error json
            return '[request]';
          }
          return value;
        },
        2);
  }
};
```

Reply:
Added CustomError class